zebra-chain/src/transaction/unmined.rs

```diff
@@ -29,6 +29,24 @@ use crate::{
 
 use UnminedTxId::*;
 
+/// The minimum cost value for a transaction in the mempool.
+///
+/// Contributes to the randomized, weighted eviction of transactions from the
+/// mempool when it reaches a max size, also based on the total cost.
+///
+/// > Each transaction has a cost, which is an integer defined as:
+/// >
+/// >     max(serialized transaction size in bytes, 4000)
```

Comment:
rustdoc thinks this is a doctest, not sure if there's a better way to fix this

```suggestion
/// > max(serialized transaction size in bytes, 4000)
```